### PR TITLE
SingleLineCommentSpacingFixer - Introduction

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: src/Fixer/Comment/SingleLineCommentSpacingFixer.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+final class SingleLineCommentSpacingFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Single-line comments most have proper spacing.',
```
Comment (paulbalandan) [inline src/Fixer/Comment/SingleLineCommentSpacingFixer.php:33]:
```suggestion
            'Single-line comments must have proper spacing.',
```

Comment (SpacePossum) [reply src/Fixer/Comment/SingleLineCommentSpacingFixer.php]: thanks so much, updated :+1:

**Comment by mvorisek** [comment]
why is such fixer useful and enabled by in `@PhpCsFixer` ruleset?

example https://github.com/atk4/ui/runs/5461667530?check_suite_focus=true

we are used to put space after // when it is a "comment that should stay"

but if some "code is commented temporary", there is no space

this is a semantic convection and help the develop to understand temporary commented code vs. regular comment

**Comment by SpacePossum** [comment]
>why is such fixer useful and enabled by in @PhpCsFixer ruleset?

That is set is for this project itself and we do not follow the rule for comments that you describe. The rule is in clear conflict with your (teams) convention, but not with this project. As such, I would recommend not using the rule set for this project but to create you own config for your project.

**Pull request merged**